src/blueapi/plugins/data_writing_server.py

```diff
@@ -0,0 +1,76 @@
+import itertools
```

Comment:
This whole file is just a prototype version of the visit service, now embedded in GDA, and should be deleted!